sim/midas/src/main/scala/midas/widgets/UsesHostDRAM.scala

```diff
@@ -49,7 +49,9 @@ trait HostDramHeaderConsts {
    * to give it an independent region.
    *
    */
-  def memoryRegionName: String
+  def memoryRegionNameOpt: Option[String]
```

Comment:
Can you explain the rationale for this change. It has knock on effects for DRAM allocation.

Reply:
The `HostDramHeaderConsts` trait is used by `SerialBridgeModule`, which does not have a `memoryRegionName` if there is no target DRAM.

Reply:
I think in that case what we should do is remove the mixin from the serial bridge; and directly add an option member there. 

If we want bridges to be able to inspect the dram allocation map, i think we can push something into the parameters instance before FPGATop module instantiation.

Reply:
Ill wait for the rest of the CY PR though.

Reply:
CY PR is here: https://github.com/ucb-bar/chipyard/pull/682